tests/unit/openwrt/golden/ipq806x/root/bin/config-version.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+usage(){
+  cat << EOF
+usage: "$(basename "$0")" [OPTIONS] ARGS
+Set OpenWRT config to specific version
+OPTIONS:
+  -h      Show this message
+  -c      Configuration version [default: 0]
+EXAMPLES:
+  To set config version 1:
+      "$(basename "$0")" -c 1
+EOF
+}
+
+CONFVER=0
+FILES=(wireless network)
+CONFROOT='/etc/config'
+
+while getopts "h:c:" OPTION
+do
+  case $OPTION in
+    c )
+      CONFVER=$OPTARG
+      ;;
+    h )
+      usage
+      exit 0
+      ;;
+    \? )
+      usage
+      exit 1
+      ;;
+  esac
+done
+shift $((OPTIND-1))
+
+RELOAD=0
+
+for file in ${FILES[*]}
```

Comment:
This should be done in two passes, if _any_ of the files for the config don't exist, we don't want to switch any of them

Reply:
Good idea, simple enough to add

Reply:
Agreed. Let's hold get that implemented before we merge this PR

Reply:
@davidelang heres the check: https://github.com/socallinuxexpo/scale-network/pull/203/files#diff-009c980269baa7501260e6c3f1b9e298R51-R53